### ISSUE TITLE
CSS issue with 'outline-color': inconsistent behaviour between browsers

### ISSUE THREAD

**Issue by JohnXLivingston** [conversejs/converse.js#3307]
**Describe the bug**

I found some inconsistencies between Firefox And Chrome, concerning the outline-color of textareas.

TL;DR: https://stackoverflow.com/questions/67667161/is-css-outline-color-property-is-not-working-in-firefox-e-g-inputfocus-ou
On firefox, if you don't set `outline-style`, firefox adds a blue outline. 

Here is what i get on my ConverseJS-based chat.

|browser|without focus| with focus|
|--|--|--|
|Firefox|![image](https://github.com/conversejs/converse.js/assets/38844060/05429038-9c29-4ac5-aefa-1620e332eb08)|![image](https://github.com/conversejs/converse.js/assets/38844060/57751315-92df-4329-ac16-625c7da1e966)|
|Chromium|![image](https://github.com/conversejs/converse.js/assets/38844060/b78e928d-408e-413b-9e89-5c56d459b71b)|![image](https://github.com/conversejs/converse.js/assets/38844060/6b305b5f-1acf-4eac-8138-353faa51103b)|

If I add this CSS in firefox:

```CSS
textarea {
  outline-style: solid;
  outline-width:0px;
}
textarea:focus {
  outline-width:2px;
}
```

I have something close to the chrome rendering. Just the border has no radius (see bellow).
But adding these rules has the same effects on chromium, and removes the radius effect.

|browser|without focus| with focus|
|--|--|--|
|Firefox|![image](https://github.com/conversejs/converse.js/assets/38844060/aafb2cc0-3f44-4e83-b6a5-43bc18db65c3)|![image](https://github.com/conversejs/converse.js/assets/38844060/91e47fa0-d66f-4966-ac77-1e2fd5167e04)|
|Chromium|![image](https://github.com/conversejs/converse.js/assets/38844060/32375748-df56-4cb1-b6da-3bb94b2c2cc4)|![image](https://github.com/conversejs/converse.js/assets/38844060/9eac3a62-970b-4ad3-92f5-3bdedd654339)|

Note: if we don't set `outline-width`, firefox and chrome have not the same behaviour when the field has not the focus: firefox displays a 1px outline, chrome nothing.

So, i will make a PR adding something consistent on textarea.

**Comment by JohnXLivingston** [comment]
I think i will make a 1px outline when the textarea has not the focus. I had a user who complained because we can't clearly differenciate the message field: https://github.com/JohnXLivingston/peertube-plugin-livechat/issues/311

**Comment by JohnXLivingston** [comment]
Here is the PR: https://github.com/conversejs/converse.js/pull/3308/

**Comment by jcbrand** [comment]
Thanks, I've merged your PR manually.

**Comment by JohnXLivingston** [comment]
Thanks @jcbrand :)
